dash_cytoscape/package.json

```diff
@@ -16,8 +24,8 @@
     "build:all": "npm run build:js-all && npm run build:py",
     "build:all-activated": "(. venv/bin/activate || venv\\scripts\\activate && npm run build:all)"
   },
-  "author": "The Plotly Team",
-  "author-email": "xinghan@plot.ly",
+  "author": "The Plotly Team <cytoscape@plot.ly>",
```

Comment:
Seems like those changes overlap with PR #70 . I think it'll be nice to merge the other one first before bring those changes to `package.json`.